build.gradle.kts

```diff
@@ -35,9 +35,10 @@ kotlin {
       }
 
       linuxX64()
+//      linuxArm64()  //
```

Comment:
@sksamuel I think is required for linuxArm64, but we'd need Koin project to add this into their build targets as well.